paddle/fluid/operators/jit/gen/seqpool.h

```diff
@@ -193,7 +193,7 @@ class SeqPoolJitCode : public JitCode {
   }
 
  private:
-  float ALIGN32_BEG fp_h_[1] ALIGN32_END;
+  float fp_h_[1];
```

Comment:
no need remove this since #15626

Reply:
I still have the question, what is this alignment for? What does it achieve here?